src/interactions/fene_bond.jl

```diff
@@ -0,0 +1,61 @@
+export FENEBond
+
+@doc raw"""
+    FENEBond(; k, r0, ϵ, σ)
+
+A finitely extensible nonlinear elastic (FENE) bond between two atoms ([Kremer, Grest, J Chem Phys, 92, 5057 (1990)](https://doi.org/10.1063/1.458541)). The potential energy is given by
+
+```math
+U(r) = -\frac{1}{2} k r^2_0 \ln (1 - (\frac{r}{r_0})^2) + U_{\text{WCA}}(r)
+```
+where the WCA contribution is given by
+```math
+U_{\text{WCA}}(r) =
+    \begin{cases}
+      4\epsilon \big[ (\frac{\sigma}{r})^{12} - (\frac{\sigma}{r})^6 \big] + \epsilon & r < 2^{1/6}\sigma\\
+      0 & r \geq 2^{1/6}\sigma\\
+    \end{cases}       
+```
+"""
+struct FENEBond{D, K} <: SpecificInteraction
+    k::K
+    r0::D
+    ϵ::K
+    σ::D
+end
+
+FENEBond(; k=1, r0=1.5, ϵ=1.0, σ=1.0) = FENEBond{typeof(r0), typeof(k)}(k, r0, ϵ, σ)
```

Comment:
Defaults should have relevant units in general.

Reply:
Just removed the default values.